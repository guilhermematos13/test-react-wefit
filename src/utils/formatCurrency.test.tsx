import { formatCurrency } from "./formatCurrency";

describe("Format Currency Function", () => {
  it("should be test format currency", () => {
    expect(formatCurrency(10)).toMatchSnapshot();
  });
});
