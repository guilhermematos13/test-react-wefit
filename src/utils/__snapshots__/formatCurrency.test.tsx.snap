// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format Currency Function should be test format currency 1`] = `"R$Â 10,00"`;
